bflib/src/jtac.rs

```diff
@@ -698,6 +719,114 @@ impl Jtac {
         Ok(())
     }
 
+    pub fn alcm_mission(&mut self, db: &Db, lua: MizLua, gid: &GroupId, n: u8) -> Result<()> {
+        match self.target.as_mut() {
+            None => bail!("no target"),
+            Some(target) => {
+                let name = db.group(gid)?.name.clone();
+                let apos = db.group_center(gid)?;
+                let pos = Vector2::new(target.pos.x, target.pos.z);
+                let expend = match n {
+                    1 => WeaponExpend::One,
+                    2 => WeaponExpend::Two,
+                    4 => WeaponExpend::Four,
+                    12 => WeaponExpend::All,
+                    _ => bail!("invalid expend {n}"),
```

Comment:
I guess in the menu you can make this make some sense ... I guess. But the chat interface is going to make chicken heads explode.